src/components/Menu/index.js

```diff
@@ -1,35 +1,51 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { css } from 'emotion'
-import { Navbar, Nav } from 'react-bootstrap'
 import Link from 'gatsby-link'
 import { navigate } from 'gatsby'
 import { styles } from './styles'
 import MenuDropdownLink from '../MenuDropdownLink'
-import logo from '../../images/logo_large.png'
+//import logo from '../../images/logo_large.png'
 
 const Menu = ({ location }) => {
   let pathname = null
 
   if (location) {
     pathname = location.pathname
-  } 
+  }
   return (
-    <Navbar fluid={true} className={css(styles.menuContainer)}>
-      <Navbar.Header className={css(styles.brand)}>
-        <Navbar.Brand className={css(styles.logoWrap)}>
-          <Link to="/selected">
-            <img className={css(styles.logoImage)} src={logo} alt="HAHA Studio" />
```

Comment:
@Alphascythian we need to bring back the logo instead of text, but this time try to implement via gatsby image component